### PR TITLE
Add server unavailable to inspector

### PR DESCRIPTION
**Pull request by flozia**
Adds the ability to timeout all servers of a city through the Mozilla VPN Inspector using `force_server_unavailable {countryCode} {cityCode}`.

### CODE REVIEW

**Review by strseb — APPROVED** [review]

File: src/ui/main.qml [hunk 1]
```diff
@@ -441,6 +441,13 @@ Window {
         id: serverUnavailablePopup
     }
 
+    Connections {
+        target: VPNController
+        function onReadyToServerUnavailable() {
+            serverUnavailablePopup.open();
+        }
+    }
```
Comment (flozia) [inline src/ui/main.qml:449]: I think this part was lost during a rebase with master. Also happy to create a separate PR to add this line back in.

File: src/controller.cpp [hunk 2]
```diff
@@ -445,6 +445,16 @@ void Controller::handshakeTimeout() {
   activateInternal();
 }
 
+void Controller::setCooldownForAllServersInACity(const QString& countryCode,
+                                                 const QString& cityCode) {
+  logger.debug() << "Set cooldown for all servers in a city";
```
Comment (strseb) [inline src/controller.cpp:450]: imho, we should have a `Q_ASSERT(!Constants::inProduction())` - so everyone knows this is for debugging / testing

**Review by strseb — APPROVED** [review]

File: src/controller.cpp (hunk 2 above)
Comment (flozia) [reply src/controller.cpp:450]: That is a good point! Unfortunately we’ll need to be able to use the method in production as well so QA can test the feature — I’ll add a comment for now if that is OK.

**Review by oskirby — APPROVED** [review]
Looks good!

**Pull request merged**